README.md

```diff
@@ -11,34 +11,32 @@ EvaP is the course evaluation system used internally at Hasso Plattner Institute
 
 For the documentation, please see our [wiki](https://github.com/e-valuation/EvaP/wiki).
 
+## Development Setup
 
-## Installation (for Development)
+We use [nix](https://nixos.org/) to manage the development environment.
 
-The easiest setup using [Vagrant](https://www.vagrantup.com) is shown here.
-
-1. Install [git](https://git-scm.com/downloads), [Vagrant](https://www.vagrantup.com/downloads.html), and one of [VirtualBox](https://www.virtualbox.org/wiki/Downloads) (recommended) or [Docker](https://docs.docker.com/engine/install/) (for ARM systems).
-
-2. Run the following commands on the command line to clone the repository, create the Vagrant VM and run the Django development server.
-   * If you are familiar with the fork-based open source workflow, create a fork and clone that (using SSH if you prefer that).
-
-   * Windows users: We have observed [weird](https://www.github.com/git-for-windows/git/issues/4705) [behavior](https://www.github.com/git-for-windows/git/issues/4704) with SSH in Git Bash on Windows and thus recommend using PowerShell instead.
-
-   * To use Docker, replace `vagrant up` with `vagrant up --provider docker && vagrant provision`.
-
-   ```bash
+1. Windows only: Install the Windows Subsystem for Linux (WSL) using `wsl --install -d Ubuntu-24.04` (you may have to restart your computer). Enter the WSL environment using the `wsl` command or by selecting the entry in the [Windows Terminal](https://aka.ms/terminal). On your first entry, you need to choose a username and password - anything works (for example: username "evap", password "evap").
+2. Install [git](https://git-scm.com/downloads) and run the following commands to clone and enter the EvaP repository:
+   ```
    git clone --recurse-submodules https://github.com/e-valuation/EvaP.git
    cd EvaP
-   vagrant up
-   vagrant ssh
    ```
-   and, after the last command opened an SSH session in the development machine:
-   ```bash
+3. Install [nix](https://nixos.org/). For Linux and WSL, we recommend using our installation script by running `./nix/setup-nix`. For MacOS, we recommend using the [Determinate Nix Installer](https://install.determinate.systems/).
+4. Start the needed background services for EvaP:
+   ```
+   nix run .#services-full -- --detached
```

Comment:
Name for "services-full" (postgres, redis, run npm ci, run manage.py migrate etc.) is up for debate by the way.

Secondly, we can also choose one key as the default for just plain `nix run -- --detached`. Canonically, this would run EvaP and not the background services, but we could choose to be uncanonical here. I think the `.#services` (or so) is fine for now